backend/book-app/src/test/java/com/karankumar/bookproject/book/controller/UserControllerTest.java

```diff
@@ -40,17 +44,7 @@
 import static org.assertj.core.api.SoftAssertions.assertSoftly;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.doThrow;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
-
-import org.springframework.test.context.ActiveProfiles;
-
-import javax.mail.MessagingException;
-import javax.validation.ConstraintViolationException;
+import static org.mockito.Mockito.*;
```

Comment:
As per our style guide, we are not using wildcard imports, so could you please undo this?

Reply:
sorry, i'll do it